Dockerfile

```diff
@@ -1,6 +1,12 @@
-FROM gradle:jdk16@sha256:d31e12d105e332ec2ef1f31c20eac6d1467295487ac70e534e3c1d0ae4a0506e AS build
-COPY --chown=gradle:gradle . /home/gradle/src
+FROM gradle:7.6.3-jdk17 AS build
```

Comment:
Don't mind if it's in this change or another, but we should set up dependabot for docker